newIDE/app/src/Profile/ProfileDetails.js

```diff
@@ -17,25 +19,61 @@ type Props = {
 
 export default ({ profile, onEditProfile }: Props) => {
   return profile ? (
-    <Column>
-      <Line alignItems="center">
-        <Avatar src={getGravatarUrl(profile.email || '', { size: 40 })} />
-        <Spacer />
-        <Text size="title">{profile.username}</Text>
-      </Line>
-      <Line alignItems="center">
-        <Text>
-          <Trans>You are connected as {profile.email}</Trans>
-        </Text>
-      </Line>
-      <Line justifyContent="center">
-        <RaisedButton
-          label={<Trans>Edit my profile</Trans>}
-          primary
-          onClick={onEditProfile}
-        />
-      </Line>
-    </Column>
+    <I18n>
+      {({ i18n }) => (
+        <Column>
+          <Line alignItems="center">
+            <Avatar src={getGravatarUrl(profile.email || '', { size: 40 })} />
+            <Spacer />
+            <Text
+              size="title"
+              style={{
+                color: profile.username ? 'black' : '#888',
+              }}
+            >
+              {profile.username ||
+                i18n._(t`Edit your profile to pick a username!`)}
+            </Text>
+          </Line>
+          <Line>
+            <TextField
+              value={profile.email}
+              readOnly
+              fullWidth
+              floatingLabelText={<Trans>Email</Trans>}
+              floatingLabelFixed={true}
+            />
+          </Line>
+          <Line>
+            <TextField
+              value={
+                profile.description ||
+                i18n._(
+                  t`No bio defined. Edit your profile to tell us what you are using GDevelop for!`
+                )
+              }
+              readOnly
+              fullWidth
+              multiline
+              floatingLabelText={<Trans>Bio</Trans>}
+              floatingLabelFixed={true}
+              inputStyle={{
+                color: profile.description ? 'black' : '#888',
+              }}
```

Comment:
Here, prefer passing a hint text when the value is empty :) 
This way, no need for inputStyle.

Reply:
I thought that there was a problem with hint text, but not at all indeed. Will use it